service-deployment/service.yaml

```diff
@@ -72,6 +72,11 @@ imagelist:
       - dockerfile
     prerequisite: None
 
+  gpu-exporter:
+    templatelist:
+      - dockerfile
+    prerequisite: drivers
```

Comment:
Why gpu-exporter depends on drivers image? Here is the image list, not service list. If your image doesn't depend on drivers, you should set None here.

Reply:
Solved #Resolved